functions/scripts/email-bulk-delivery-volunteers.js

```diff
@@ -0,0 +1,101 @@
+const functions = require('firebase-functions')
+const _ = require('lodash')
+const sgMail = require('@sendgrid/mail');
+const showdown  = require('showdown')
+
+sgMail.setApiKey(functions.config().sendgrid.api_key);
+
+const { INTAKE_TABLE, VOLUNTEER_FORM_TABLE, getRecord, getRecordsWithStatus } = require('../airtable')
+
+async function getBulkDeliveryConfirmedTickets() {
+  return await getRecordsWithStatus(INTAKE_TABLE, 'Bulk Delivery Confirmed')
+}
+
+function clusterTickets(tickets) {
+  return _.groupBy(tickets, ([, fields,]) => {
+    if (fields['Bulk Cluster'].length !== 1) {
+      throw new Error(`Ticket ${fields} does not have one Bulk Cluster`)
+    }
+    return fields['Bulk Cluster'][0]
+  })
+}
+
+async function getDeliveryVolunteerInfo(cluster, tickets) {
+  const volunteerIds = _.union(
+    _.flatMap(tickets, ([, fields,]) => {
+      if (!fields.deliveryVolunteer || fields.deliveryVolunteer.length !== 1) {
+        throw new Error(`Ticket ${fields.ticketID} doesn't have exactly one volunteer: ${fields.deliveryVolunteer}`)
+      }
+      return fields.deliveryVolunteer
+    })
+  )
+  if (volunteerIds.length !== 1) {
+    throw new Error(`Cluster ${cluster} doesn't have exactly one delivery volunteer: ${volunteerIds}`)
+  }
+  return await getRecord(VOLUNTEER_FORM_TABLE, volunteerIds[0])
+}
+
+function renderEmail({ cluster, volunteer }) {
+  var email = `
+  Thank you for volunteering to deliver groceries to our neighbors!
+
+  We've assigned you the following tickets: ${_.join(_.map(cluster, 'ticketID'), ', ')}
+
+  This coming Saturday, please come to Brooklyn Packers TODO(address) between 1pm and 3pm to pick up boxes.
```

Comment:
```suggestion
  This coming Saturday, please come to the warehouse at **221 Glenmore Ave, Gate 4 at 1pm**, to pick up your deliveries. Since there are perishables in the deliveries, you'll need to deliver them immediately after pickup.
```